src/time_travel/time_travel.py

```diff
@@ -0,0 +1,69 @@
+"""Mocking interface for python time libraries."""
+
+from datetime_patcher import DatetimePatcher
+from time_patcher import TimePatcher
+
+
+class TimeMachineClock(object):
+    """Unifing class for clock types."""
+    
+    def __init__(self, start_timestamp=0.0):
+        """Initialize a unifing clock."""
+        self._timestamp = start_timestamp
+        
+    def get_timestamp(self):
```

Comment:
To solve early evaluation of the property when passing it to the mock's side_effect.
Moved the function to the relevant patcher.